packages/aws-cdk/lib/init-templates/sample-app/python/setup.template.py

```diff
@@ -6,7 +6,7 @@
 
 
 setuptools.setup(
-    name="%name%",
+    name="%name.PythonModule%",
```

Comment:
From a user's standpoint, is `PythonModule` something that would be idiomatic for users of Python? 

I'm curious if we could simplify the naming here (and elsewhere in this PR). It would be good to add the rationale behind the naming for posterity in any case.

Reply:
I didn't want to make that decision. I simply copied what was already being done in other python templates.
Who else can we ask for comments here?

Reply:
@shivlaks `%name.PythonModule%` is a token here, & gets replaced by the actual module name (generated from the folder name). So, python users won't be seeing this in the projects they generate from these templates.
If we still decide to change the name of this token, I believe that'd be out of scope for this PR.

Reply:
thanks for following up, seems fine as it is